relay/cmd/main.go

```diff
@@ -76,7 +73,6 @@ var (
 func init() {
 	_ = util.InitLog("trace", "console")
```

Comment:
Do we still need this?

Reply:
I think so. If we have any issues before line 104. (where we process the log parameters) then I still want to use our logging format.